string[] joint_names
float64[] joint_values
---
std_msgs/Empty empty
